networking/src/ApiClient.ts

```diff
@@ -90,6 +90,22 @@ export class ApiClient implements IApiClient {
 
     }
 
+    async updateGameSessionFooter(id: string, gameState: GameSessionState, nextQuestion: number, phaseOneTimeReset: number, phaseTwoTimeReset: number): Promise<IGameSession> {
```

Comment:
What's the purpose of `phaseTwoTimeReset` and `phaseOneTimeReset`? Is teacher able to change the default values for phase one and phase two timers during the game session?